static/js/services/vulnerability-service.js

```diff
@@ -570,7 +1044,8 @@ angular.module('quay').factory('VulnerabilityService', ['Config', 'ApiService',
           'severity': 'low'
         }
       ]
-    }
+    },
+
```

Comment:
I don't think this trailing comma is allowed, but maybe the rules are different in javascript?

Reply:
Yes, JavaScript allows trailing commas (ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas)